skl2onnx/algebra/complex_functions.py

```diff
@@ -61,14 +61,14 @@ def _onnx_squareform_pdist_sqeuclidean(X, dtype=None, op_version=None,
     return node[1]
 
 
-def onnx_cdist(X, Y, metric='sqeuclidean', dtype=None,
+def onnx_cdist(XA, XB, metric='sqeuclidean', dtype=None,
```

Comment:
Any reason of having the names in capitals, pep8 recommends variable names to be in lower case.

Reply:
I reused the same names scipy is using.